cli/azd/pkg/templates/template.go

```diff
@@ -12,3 +20,28 @@ type Template struct {
 	// or "{repo}" for GitHub repositories under Azure-Samples (default organization).
 	RepositoryPath string `json:"repositoryPath"`
 }
+
+// Display writes a string representation of the template suitable for display.
+func (t *Template) Display(writer io.Writer) error {
+	tabs := tabwriter.NewWriter(
+		writer,
+		0,
+		output.TableTabSize,
+		1,
+		output.TablePadCharacter,
+		output.TableFlags)
+	text := [][]string{
+		{"RepositoryPath", ":", t.RepositoryPath},
+		{"Name", ":", t.Name},
+		{"Description", ":", t.Description},
```

Comment:
Nit: PR description says we're "Removing" description, but looks like we just have an alternate display.